net-misc/frr/frr-7.2.1.ebuild

```diff
@@ -0,0 +1,157 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools pam systemd
+
+DESCRIPTION="The FRRouting Protocol Suite"
+HOMEPAGE="https://frrouting.org/"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~x86 amd64"
+
+SRC_URI="https://github.com/FRRouting/frr/archive/${P}.tar.gz"
+
+IUSE="
+	babel +bfd +bgp doc eigrp +fabric fpm +ipv6 +isis +ldp nhrp +ospf ospfapi
+	pam pbr +pim realms rip +rpki +rtadv sanitize snmp systemd vrrp"
+
+REQUIRED_USE="
+	rpki? ( bgp )
+"
+
+COMMON_DEPEND="
+	=net-libs/libyang-0.16.104
+	dev-lang/python
+	dev-libs/json-c
+	nhrp? ( net-dns/c-ares )
+	pam? ( sys-libs/pam )
+	rpki? ( >=net-libs/rtrlib-0.6.3[ssh] )
+	snmp? ( net-analyzer/net-snmp )
+	sys-libs/libcap
+	sys-libs/readline
+"
+
+BDEPEND="
+	${COMMON_DEPEND}
+	doc? ( dev-python/sphinx )
+	sys-devel/flex
+	virtual/yacc
+"
+
+DEPEND="
+	${COMMON_DEPEND}
+"
+
+RDEPEND="
+	!!net-misc/quagga
+	${DEPEND}
+	acct-user/frr
+	app-shells/bash
+"
+
+# FRR tarballs have weird format.
+S="${WORKDIR}/frr-${P}"
+
+src_prepare() {
+	default_src_prepare
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--with-pkg-extra-version="-${PR}-gentoo" \
+		--enable-configfile-mask=0640 \
+		--enable-logfile-mask=0640 \
+		--prefix=/usr \
+		--sysconfdir=/etc/frr \
+		--libdir=/usr/lib/frr \
+		--sbindir=/usr/lib/frr \
+		--libexecdir=/usr/lib/frr \
+		--localstatedir=/var/run/frr \
+		--with-moduledir=/usr/lib/frr/modules \
+		--enable-exampledir=/usr/share/doc/${PF}/samples \
+		--enable-user=frr \
+		--enable-group=frr \
+		--enable-vty-group=frrvty \
+		--enable-multipath=64 \
+		$(use_enable babel babeld) \
+		$(use_enable bfd bfdd) \
+		$(use_enable bgp bgpd) \
```

Comment:
needs `$(usex bgp $(use_enable rpki))` for RPKI

Reply:
Nice catch. I rebased the commits with the fix to avoid increasing commit count.

It now shows me this when I merge:
```
Fixing .la files                                                                                                       
   usr/lib/frr/modules/bgpd_bmp.la   
strip: x86_64-pc-linux-gnu-strip --strip-unneeded -N __gentoo_check_ldflags__ -R .comment -R .GCC.command.line -R .note.gnu.gold-version
   /usr/lib/frr/libfrrcares.so.0.0.0                
   /usr/lib/frr/libfrrcares.a                              
   /usr/lib/frr/libfrr.so.0.0.0                            
   /usr/lib/frr/libfrr.a
   /usr/lib/frr/modules/zebra_irdp.so          
   /usr/lib/frr/modules/bgpd_rpki.so                  
   /usr/bin/mtracebis                                   
   /usr/lib/frr/zebra
   /usr/bin/vtysh                    
   /usr/lib/frr/watchfrr                                                                                               
   /usr/lib/frr/ssd           
   /usr/lib/frr/ospf6d               
   /usr/lib/frr/ospfd                  
   /usr/lib/frr/bgpd                  
   /usr/lib/frr/isisd                                                                                                  
   /usr/lib/frr/fabricd                              
   /usr/lib/frr/ldpd                                                                                                   
   /usr/lib/frr/staticd                           
   /usr/lib/frr/modules/bgpd_bmp.so                    
   /usr/lib/frr/pimd                             
   /usr/lib/frr/bfdd
```

Notice:

```
   /usr/lib/frr/modules/bgpd_rpki.so
```

Thanks!